src/components/Accordion/AccordionCollapsible.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { includes } from 'ramda';
+import { transparentize } from 'polished';
+
+import { getColor, pxToRem } from '../../utils/helpers';
+import { FlexContainer } from '../FlexContainer';
+import { Icon } from '../Icon';
+import { IconTypes, SSCIconNames } from '../Icon/Icon.enums';
+import { Text } from '../typography';
+import { TextSizes } from '../typography/Text/Text.enums';
+import { AccordionCollapsibleProps } from './Accordion.types';
+
+const Header = styled(FlexContainer)`
+  width: 100%;
+  height: ${pxToRem(60)};
+  padding: ${pxToRem(0, 13)};
+  cursor: pointer;
+`;
+
+const Content = styled(Text)`
+  padding: ${pxToRem(24, 40)};
+  border-top: 1px dashed ${getColor('graphiteHB')};
+`;
+
+const Container = styled.div<{ isOpen: boolean }>`
+  width: 100%;
+  ${({ isOpen }) =>
+    isOpen &&
+    css`
+      background: ${getColor('graphite5H')};
+      box-shadow: 0px 2px 6px 0px ${transparentize(0.85, '#000')};
+    `}
+`;
+
+const StyledIcon = styled(Icon).withConfig<{ isRotated: boolean }>({
+  shouldForwardProp: (property) => !includes(property, ['isRotated']),
+})`
+  margin-right: ${pxToRem(13)};
+  transition: transform 200ms;
+  ${({ isRotated }) => isRotated && 'transform: rotate(90deg);'}
+  height: ${pxToRem(11)};
+  width: ${pxToRem(11)};
+`;
+
+const AccordionCollapsible: React.FC<AccordionCollapsibleProps> = ({
+  children,
+  className,
+  handleHeaderClick,
+  isOpen = false,
+  index,
+  title,
+}) => (
+  <Container className={className} isOpen={isOpen}>
+    <Header alignItems="center" onClick={() => handleHeaderClick(index)}>
+      <StyledIcon
+        isRotated={isOpen}
```

Comment:
Not sure what's @ajkl2533 take on that but I like the new transient props for this so you don't have to use shouldForwardProp - https://styled-components.com/docs/api#transient-props

Reply:
@vaclav-zeman if our styled supports it and eslint won't scream at me for invalid property name, why not 😁

Reply:
@vaclav-zeman tried it, and it works fine, so depends on @ajkl2533 whether he likes it. It is a little bit odd to put special character into prop name. But it's internal stuff on the other hand, so it should not irritate anyone.

Reply:
I'm totally fine with using transient props for internal API. I would rather not to expose it to end-users tho.